### DISCUSSION TITLE
Publish to microsoft store in msix format

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#602] [Ideas]
Example benefits of msix format:
https://github.com/felixrieseberg/windows95/issues/221 https://learn.microsoft.com/en-us/windows/msix/overview#key-features

**Comment by Et0h** [comment]
NSIS is fit for purpose. It would take developer time to even investigate the viability of using msix - the signing requirements might be a showstopper. I have no experience with msix and a lot of experience with NSIS. As such, I don't have any plans to change installer system to msix (or anything else) in the foreseeable future.

**Reply by soredake** [reply]
About signing: https://www.advancedinstaller.com/msix-publish-microsoft-store.html

>Automatic Digital Signing - MSIX requires[ digital signing](https://www.advancedinstaller.com/msix-digital-signing.html) and, by publishing it to Microsoft Store, your app gets signed automatically. Therefore, you don’t need to buy a third-party CA (certification authority), and your app will have the credibility of being secure in case you use your own certification.
